state-transition/core/state_processor_genesis.go

```diff
@@ -71,7 +71,7 @@ func (sp *StateProcessor[
 	}
 
 	// TODO: we need to handle chain.Version vs uint32 better.
-	var blkBody BeaconBlockBodyT
+	var blkBody *ctypes.BeaconBlockBody
```

Comment:
we may consolidate these kind of methods, but I guess it's fine to clean them up later on too